prisma/schema.prisma

```diff
@@ -68,6 +68,7 @@ model Transaction {
   id        Int                    @id @default(autoincrement())
   hash      String                 @db.VarChar(255)
   amount    Decimal                @db.Decimal(24, 8)
+  confirmed Boolean                @default(true)
```

Comment:
Whats the reason to have this default to `true` instead of `false`? Seems like it could potentially bite us

Reply:
First because we were already only doing confirmed txs, but also because I think the idea of saving the transaction is mostly those that actually happened. The 'non confirmed' ones are actually the exception, because they are not really transactions, only potential transactions, so when we save them we make it clear that we are not saving a confirmed transaction and that necessarily appears on the code, since the default is `true`.

Reply:
I think you're arguing mostly from a point of code clarity, which is important. @Gattaca1 is arguing from a point of security. It's better to assume a transaction is unconfirmed in this case, I think.
Relying on our code to explicitly specify an unconfirmed transaction could result in human-error. There are 0-conf attacks that can reject an unconfirmed transaction from the mempool.
If there's a human-error that results in a false-negative (incorrectly flagging a transaction as unconfirmed), then the side-effects are less severe than a false-positive, I'd expect.